Tables/Change-font-of-table-content/Change-font-of-table-content/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Syncfusion.DocIO.DLS;
+using Syncfusion.DocIO;
+
+using (FileStream inputStream = new FileStream("Data/Input.docx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+{
+    // Open the input HTML format document.
+    using (WordDocument document = new WordDocument(inputStream, FormatType.Docx))
```

Comment:
Input is DOCX, but xml comments contains HTML.

Change the xml comment properly.

Reply:
Addressed the feedback